wiki/Ranking_Criteria/en.md

```diff
@@ -106,7 +106,7 @@ Changes to the rules and guidelines in this document are proposed and discussed
 
 #### Romanisation
 
-- **An artist's name is to be romanised in the order it is printed in the unicode field, unless an alternative order for the romanised name is provided by the artist.**
+- **An artist's name must be romanised in the order it is printed in the unicode field if there is no official romanisation or translation available.** When an official romanisation is available, this method is optional.
```

Comment:
this still reads shockingly. what is optional? why is it optional? why is the official romanisation not preferred? it should be preferred no?

what's a "method"? you mean order?

Reply:
Forcing it in a rule would be a regression as some official romanizations are extremely obscure and to require someone to find it to follow the RC. Think a single occurence on one album 12 years ago or buried agency pages. Defining what counts or not for this would be a headache in its own right.

I mean order.

Reply:
Shouldn't it still be preferred? If you want to guard against effort being expended, something like:

"If an official romanisation is readily available, it should be preferred. If not, an artist's name must be a direct romanisation of the unicode artist name (including order)"

Reply:
With the allowances in place there's a tendency to prefer what's prevalent and recognizable already, as has been the case for song titles. Your rewording would work similarly to what's currently here in the PR but...

However, that wording "should be preferred" would make the comment into its own guideline. There are currently no guideline section for romanization as all relevant parts have been treated as hard rules, formatting wise it would separate this clause from the rest of the romanization sections, making it a bit more obscure to notice in RC.

Reply:
With https://github.com/ppy/osu-wiki/pull/6079#discussion_r706344648 in mind,

Does 
```suggestion
- **When romanising an artist's name yourself, it must be romanised in the order it is printed in the unicode field.** This must be done if no official romanisation is available.
```

work? This forces no specific method when official options are given, but still leaves the option of romanising it this way open and not breaking any rules in cases where "official" is obscure and it is romanised by the mapper or modder.

Preference for official/recognizable over all should be in its own guideline as mentioned in the other comment.

Reply:
note that since you removed "or translation" and moved the part outside like that, it would imply that if there is no official romanisation but there is a translation, the artist name must be romanised anyway (and the translation discarded), which conflicts with the allowance

shuffling it around like this prevents that

```suggestion
- **When romanising an artist's name yourself and no official romanisation is available, it must be romanised in the order it is printed in the unicode field.**
```

this doesn't specify whether the the translation or the official romanisation should be picked at all or which is preferred, and isolates the clause to say exactly one thing only. if other rules after discussion will eventually specify what is preferred, this way would be cleaner.

Reply:
This reads fine, but the larger issue I touched on probably still stands (and how things are currently done sounds pretty weird). I don't have the throughput for dealing with that can of worms though.